release-team/hack/sync-bug-triage-github-project-beta.sh

```diff
@@ -0,0 +1,228 @@
+              repository(owner: $org, name: $repo) {
+                description
+                url
+                milestones(states: [OPEN],first:1, query: $milestone) {
+                  nodes{
+                    issues(states:[OPEN], first: 200){
+                      nodes{
+                        id
+                      }
+                      pageInfo{
+                        hasNextPage
+                        endCursor
+                      }
+                    }
+                  }
+                  pageInfo{
+                    hasNextPage
+                    endCursor
+                  }
+                }
+              }
+            }' -f org=${ORGANIZATION} -f repo=${REPOSITORY} -f milestone="${MILESTONE}")"
+
+no_of_issues=$(jq ".data.repository.milestones.nodes[].issues.nodes | length" <<< "${MILESTONE_ISSUES_JSON}")
+issues_loop_index_range=$(( no_of_issues - 1 ))
+
+for index in $(seq 0 $issues_loop_index_range);
+do
+	issue_id=$(jq ".data.repository.milestones.nodes[].issues.nodes[$index].id" <<< "${MILESTONE_ISSUES_JSON}")
+	milestone_issue_ids+=("${issue_id}")
+done
+
+## Fetch PRs
+MILESTONE_PRS_JSON="$(gh api graphql --paginate -f query='
+            query($org: String!, $repo: String!, $milestone: String!){
+              repository(owner: $org, name: $repo) {
+                description
+                url
+                milestones(states: [OPEN],first:1, query: $milestone) {
+                  nodes{
+                    pullRequests(states:[OPEN], first: 200){
+                      nodes{
+                        id
+                      }
+                      pageInfo{
+                        hasNextPage
+                        endCursor
+                      }
+                    }
+                  }
+                  pageInfo{
+                    hasNextPage
+                    endCursor
+                  }
+                }
+              }
+            }' -f org=${ORGANIZATION} -f repo=${REPOSITORY} -f milestone="${MILESTONE}")"
+
+no_of_prs=$(jq ".data.repository.milestones.nodes[].pullRequests.nodes | length" <<< "${MILESTONE_PRS_JSON}")
+pr_loop_index_range=$(( no_of_prs - 1 ))
+
+for index in $(seq 0 $pr_loop_index_range);
+do
+  pr_id=$(jq ".data.repository.milestones.nodes[].pullRequests.nodes[$index].id" <<< "${MILESTONE_PRS_JSON}")
+	milestone_pr_ids+=("${pr_id}")
+done
+
+# Fetch Project metadata
+echo -e "[INFO] Getting metadata for the Bug Triage GitHub Beta Project with ID: ${PROJECT_NUMBER}"
+
+PROJECT_ID=$( get_field_ids_from_github_beta_project "project-id")
+TYPE_FIELD_ID=$( get_field_ids_from_github_beta_project "type-id")
+ISSUE_OPTION_ID=$( get_field_ids_from_github_beta_project "issue-id")
+PR_OPTION_ID=$( get_field_ids_from_github_beta_project "pr-id")
+
+
+# Add data into the Project Board
+echo  '[INFO] Feeding data from k/k issues into the GitHub Project Beta'
+
+for index in $(seq 0 ${issues_loop_index_range});
```

Comment:
This would also need to account for the pagination issue I mentioned above.

Reply:
addressed. Thank you!